client/blocks/edit-gravatar/style.scss

```diff
@@ -112,3 +112,27 @@
 .edit-gravatar__unverified-user .gravatar {
 	opacity: 0.33;
 }
+
+.edit-gravatar .drop-zone {
+	border-radius: 50%;
+	width: 150px;
+	height: 150px;
+	border: 0;
+}
+
+.edit-gravatar .drop-zone__content {
+	font-size: 1em;
+}
+
+.edit-gravatar .drop-zone__content-text {
+	padding: 0 1em;
+}
+
+.edit-gravatar .drop-zone.is-active ~ .edit-gravatar__label-container {
```

Comment:
Not that I can see. `<DropZone/>` is generally added as a sibling to other elements, and the `is-active` state is contained within DropZone, so `~` is needed to get to the sibling label we want to hide. Wrapping the label component inside `DropZone` doesn't work since DropZone has `visibility:none` for the parent element unless it's active, hiding the label when we don't want it hidden.